### PR TITLE
Méli-mélo Charts: mobile table, canvas container

### PR DESCRIPTION
**Pull request by Garneauma**
Adding the following:
- Ability to display the table by default using the attribute "data-chart-details-open".
- Wrapping the <canvas> element in a container to allow for custom styling.
- Adding a <div class="table-responsive"> around the table to fix overflow issue on mobile.

### CODE REVIEW

**Comment by duboisp** [comment]
Pre-approve upon review and local testing to confirm this remain inside expérimental project checklist (méli-mélo project).

File: méli-mélo/2024-02-charts/js/charts.js [hunk 1]
```diff
@@ -32,8 +32,10 @@ var componentName = "chart",
 				tableObserver,
 				elmChartType, supportedChartTypes, defaults,
 				chartType, chartOpts, chartData,
-				chartCntnr, canvasElm, detailsElm,
-				i18n, i18nText;
+				chartCntnr, canvasElm, tableContainer,
+				i18n, i18nText,
+				detailsOpen = "",
+				chartCntnrClass = "wb-chart-container";
```
Comment (duboisp) [inline méli-mélo/2024-02-charts/js/charts.js:38]: Will need more details how this use because I don't see it's function.

**Review by duboisp — CHANGES_REQUESTED** [review]

File: méli-mélo/2024-02-charts/js/charts.js (hunk 1 above)
Comment (Garneauma) [reply méli-mélo/2024-02-charts/js/charts.js]: As per our discussion, I have removed the class, but kept the `<div>`.

**Review by duboisp — APPROVED** [review]
Reviewed and tested locally, This change is conform to our méli-mélo feature checklist.

Note: This PR do include a change to our one of our main index file "_data/sites.json" and I confirm those update is accurate. Considering that is just an utilities file, there is no issue to include it with this PR.

**Pull request merged**